src/workspace-data/data-table/wds/WDSContent.test.ts

```diff
@@ -215,4 +216,21 @@ describe('WDSContent', () => {
       expect(deleteColumnMock).toHaveBeenCalledWith(expect.any(AbortSignal), 'marble', 'color');
     });
   });
+
+  describe('edit field icon is present based on type', () => {
+    it('is displayed when editable is true and type is supported', async () => {
+      const { props } = setup({
+        ...defaultSetupOptions,
+        props: { ...defaultProps, editable: true },
+        features: { ...defaultFeatures, supportsEntityUpdating: true, supportEntityUpdatingTypes: ['string'] },
+      });
+
+      // Act
+      await act(() => {
+        render(h(WDSContent, props));
+      });
+
+      // todo: need to figure out how to get the right column to check here
```

Comment:
would appreciate help here - a bit confused how I make it easy to figure out which parts of the table I need to query to see if the icon is disabled or not (per column)